docs/sources/configure/metrics-traces-attributes.md

```diff
@@ -251,21 +251,66 @@ In this example, only the first line is used and trimmed to prevent white space
 
 ## Extra group attributes
 
-Beyla supports adding custom attributes to metrics by configuring `extra_group_attributes`. You need to add the desired group and an array of attributes to be added to this group.
+Beyla allows you to enhance your metrics with custom attributes using the extra_group_attributes configuration. This gives you the flexibility to include additional metadata in your metrics, beyond the standard set.
 
-Currently only `app_kube` group is supported, which characterises attributes specific to the cluster in k8s, e.g. container name, pod name, namespace name, pod uid and others.
+To use this feature, specify the group name and the list of attributes you want to include in that group.
 
-For example:
+Currently, only the `k8s_app_meta` group is supported. This group contains Kubernetes-specific metadata such as pod name, namespace, container name, pod UID, and more.
+
+Example configuration:
 
 ```yaml
 attributes:
   kubernetes:
     enable: true
     resource_labels:
-      app: ["app"]
+      k8s.app.version: ["k8s.app.version"]
   extra_group_attributes:
-    app_kube: ["app"]
+    k8s_app_meta: ["k8s.app.version"]
```

Comment:
Nice! Now I understand it much better, thanks!

I see that `k8s.app.version` is repeated 3 times. What do you think about a more opinionated configuration? Something like:

```yaml
attributes:
  kubernetes:
    enable: true
  extra_group:
    k8s_app_meta: ["k8s.app.version"]
```
This automatically assumes that there is an existing `resource.opentelemetry.io/k8s.app.version` (and maybe `"app.kubernetes.io/k8s.app.version"`?)

Reply:
I agree, it's done